lib/rails_admin_import/config/sections/import.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_admin/config/sections/base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.